cmd/translate/service/service.go

```diff
@@ -139,6 +158,7 @@ func init() {
 	rootCmd.PersistentFlags().Uint("port", 8080, "port to run service on") //nolint:gomnd
 	rootCmd.PersistentFlags().String("host", "0.0.0.0", "host to run service on")
 	rootCmd.PersistentFlags().String("db", "badgerdb", repo.Usage())
+	rootCmd.PersistentFlags().String("translator", "", fuzzy.Usage())
```

Comment:
I wonder if we can use `none` instead of the empty string `` as the default value.